server/aws-lsp-codewhisperer/src/language-server/chat/chatEventParser.ts

```diff
@@ -40,6 +40,10 @@ export class ChatEventParser implements ChatResult {
     static mapReferenceData(reference: Reference): ReferenceTrackerInformation {
         return {
             ...reference,
+            recommendationContentSpan: {
+                start: reference.recommendationContentSpan?.start || 0,
+                end: reference.recommendationContentSpan?.end || 0,
```

Comment:
Set defaults, as `Reference.recommendationContentSpan` interface was changed, making `start` and `end` fields optional.